tests/test_cli.py

```
@@ -113,4 +114,88 @@ def test_cli_run_missing_config():
+    """Test migrating a SQL file."""
+    input_file = Path(__file__).parent.parent / 'data_files' / 'temoa_basics_0.sql'
+    output_file = tmp_path / 'migrated.sql'
+    args = ['migrate', str(input_file), '--output', str(output_file)]
+    result = runner.invoke(app, args)
+
+    assert result.exit_code == 0, f'Migration failed: {result.exception}'
+    assert 'SQL migration completed' in result.stdout
+    assert output_file.exists()
+
+
+def test_cli_migrate_sql_file_auto_output(tmp_path):
```

Ruff (0.14.4)

Unused function argument: `tmp_path`

(ARG001)